queries/javascript/rainbow-tags-react.scm

```diff
@@ -0,0 +1,36 @@
+; This query is for highlighting just the JSX tags.
+
+;;; React.js support
```

Comment:
Please remove this comment, the file name makes it obvious that we only do React here.